src/dotnet/FileSystemHost.cs

```diff
@@ -347,6 +352,14 @@ public static Int32 SetDebugLogFile(String FileName)
         {
             return Api.SetDebugLogFile(FileName);
         }
+        /// <summary>
+        /// Return the installed version for WinFSP
```

Comment:
Please change this to read: Return the installed version for WinFsp.

(Note WinFsp vs WinFSP spelling and fullstop at end of sentence.)

Reply:
Looks good otherwise although we may want to consider renaming the method to `Version`. What do you think?

Reply:
Shouldn't methods be called with an action verb ?
In any case, we can call it version. It's fine for me.